warehouse/query-core/src/main/java/datawave/query/jexl/visitors/QueryModelVisitor.java

```diff
@@ -1,16 +1,14 @@
 package datawave.query.jexl.visitors;
 
-import com.google.common.base.Function;
 import com.google.common.collect.ArrayListMultimap;
-import com.google.common.collect.FluentIterable;
 import com.google.common.collect.Lists;
 import com.google.common.collect.Multimap;
 import com.google.common.collect.Sets;
-import datawave.query.model.QueryModel;
 import datawave.query.exceptions.DatawaveFatalQueryException;
 import datawave.query.jexl.JexlASTHelper;
 import datawave.query.jexl.JexlNodeFactory;
 import datawave.query.jexl.JexlNodeFactory.ContainerType;
+import datawave.query.model.QueryModel;
```

Comment:
OK, I am stumped in this visitor as to where any lineage problems were fixed.  Is this one purely refactoring?